web/client/translations/data.it-IT

```diff
@@ -1997,6 +1997,8 @@
             "title": "Modifica proprietà",
             "name": "Nome",
             "description": "Descrizione",
+            "createdAt": "Creato",
```

Comment:
```suggestion
            "createdAt": "Creata",
```

Reply:
Yeah it is true but during implementation, I thought disabled inputs looks better in that context just to follow the pattern of the other metadata above. Otherwise I can make it static(it's easy than the disabled input). @MV88

Reply:
i see your point, but in this case they are static information and not fields that can become editable.
so a static fields represents better what information it is showing